components/Common/Dropdown/index.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import Dropdown from '.';
+
+export default {
+  component: Dropdown,
+};
+
+const items = [...Array(10).keys()].map(item => ({
```

Comment:
Can we make the list of items be customisable? (Storybook "args" feature)

Reply:
done